### ISSUE TITLE
Add CI for running tests

### ISSUE THREAD

**Issue by capsulecorplab** [303adastra/StellarCartography#4]
should leverage recommended automation pipeline + toolchain for running Pharo tests

**Comment by RalfBarkow** [comment]
Hi Sean @capsulecorplab 

Maybe you want to take a look at [Glamorous Toolkit](https://gtoolkit.com/docs/start/)?
>  [[Releaser]] makes it possible to version deeply nested projects and repositories completely automatically.

From the Glamorous Toolkit Book:

Releaser allows for the creation of release versions for applications that use Metacello baselines to specify their loading configuration (what packages to load, their loading order and their dependencies). Releaser supports projects that use deeply nested baselines spread across many git repositories with a mix of baselines with fix dependencies and dependencies to latest versions. 

Releaser aims to be easy to debug. Several design decision, like computing all actions before executing a release and using separate models for releases and dependencies go in that direction. Releaser also uses extensively inspector views to help expose implementation details.

Currently, {{gtClass:GtRlRelease}} provides support for generating new releases. However, in the context of Glamorous Toolkit, the goal is to extend it with support for loading baselines, not only generating them. Several other tools for working with a project (like running all tests in a project, or commiting changes to multiple repositories) could be based on Releaser.

In short, to create a new release a user has to: 
 - ensure that all the code of the application that needs to be released is loaded in the current image;
 - create a {{gtClass:GtRlReleaseConfiguration}} configuration controlling various aspects of the release process;
 - run the release exporter to create new Metacello baselines that use fix versions to load dependencies. 

Releases can be done manually from the image using the inspector as a user interface, or by relying on a command line handler.

**Comment by capsulecorplab** [comment]
Thanks @RalfBarkow! I'll have to check it out!
